### ISSUE TITLE
Debugging?

### ISSUE THREAD

**Issue by ghost** [MrMEEE/ironhide#101]
Can i debug an OpenGL application that is started through optirun? When I started QtCreators Debugger it tells me the program is not in executable format. Do you know any way to get this to work or do you know another debugger which works? Thank you.

**Comment by MrMEEE** [comment]
Sry.. haven't tried that... and haven't got any good ideas.. 

Does it work if you run it without optirun??

**Comment by ghost** [comment]
Yes, without optirun the debugger starts, but since I'm creating a 3.3 OpenGL Context (which isn't supported by the intel card) i need my NVIDIA card for not getting a SegFault when the application tries to start.
